index.js

```diff
@@ -1,73 +1,143 @@
 /*
 croquis:
 
-interfaz:
-1-presiona empezar para jugar!/perdiste/ganaste
-2-Indicador de nivel
-3-boton empezar
-
-4-4 cajas con colores distintos
+    interfaz:
+    1-presiona empezar para jugar!/perdiste/ganaste
+    2-Indicador de nivel
+    3-boton empezar
+    4-4 cajas con colores distintos
 //////////////////
 ejemplo de flujo:
-1-clickeo boton empezar
-2-turno de la maquina
-3-turno del usuario
-4-gane?     repito paso 2 y 3 con + dificultad  :  perdiste 
-5-ofrecer empezar de nuevo
+    1-clickeo boton empezar
+    2-turno de la maquina
+    3-turno del usuario
+    4-gane?     repito paso 2 y 3 con + dificultad  :  perdiste 
+    5-ofrecer empezar de nuevo
 */
-document.querySelector('#empezar').onclick = function(e){
-    empezarJuego()
-}
 let secuenciaMaquina = []
 let secuenciaJugador = []
 let nivel = 0
 
+document.querySelector('#empezar').onclick = function(e){
+    empezarJuego()
+}
+
+
 const empezarJuego=()=>{
-    turnoMaquina()
+    ReinciarVariables()
+    gestionarRondas()
 }
 
-const turnoMaquina=()=>{
-    const cuadradoAleatorio = obtenerCuadradoAleatorio()
-    secuenciaMaquina.push(cuadradoAleatorio)
+const gestionarRondas=()=>{
+    const $cuadradoAleatorio = obtenerCuadradoAleatorio()
+    turnoMaquina($cuadradoAleatorio)
+    let DELAY_JUGADOR = secuenciaMaquina.length*1050
+    setTimeout(function(){
+        turnoJugador()
+    },DELAY_JUGADOR)
+}
+
+const ReinciarVariables=()=>{
```

Comment:
This should be in `camelCase` and we can name this function `reiniciar` or `reiniciarJuego`. Also try to install Prettier in Visual Studio Code: https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code to format your code.